opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/ba/ANCRegisterTestsBa.java

```diff
@@ -0,0 +1,144 @@
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameTwo))
+                .check(matches(isDisplayed()));
+    }
+    @Test
+    @Order(order = 2)
+    public void confirmANCRecordProfile() {
+        utils.openDrawer();
+        onView(ViewMatchers.withSubstring(Constants.GenericConfigUtils.anc))
+                .perform(click());
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.aboveFiveFirstNameTwo), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameTwo
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameTwo))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    @Order(order = 3)
+    public void confirmANCVisitPage() {
+        utils.openDrawer();
+        onView(ViewMatchers.withSubstring(Constants.GenericConfigUtils.anc))
+                .perform(click());
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.aboveFiveFirstNameTwo), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameTwo
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameTwo))
+                .perform(click());
+        onView(withId(R.id.textview_record_visit))
+                .perform(click());
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameTwo
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameTwo + " " + Configs.TestConfigHelper.familyName + ", "+ Configs.TestConfigHelper.aboveFiveage + " · ANC Visit"))
+                .check(matches(isDisplayed()));
+    }
+    @Test
+    @Order(order = 4)
+    public void confirmANCSearchWorks() {
+        utils.openDrawer();
+        onView(ViewMatchers.withSubstring(Constants.GenericConfigUtils.anc))
+                .perform(click());
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.aboveFiveFirstNameTwo), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameTwo
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameTwo))
+                .check(matches(isDisplayed()));
+    }
+
+    public void confirmANCPhoneNumber() throws Throwable {
+        utils.openDrawer();
+        onView(ViewMatchers.withSubstring(Constants.GenericConfigUtils.anc))
+                .perform(click());
+        onView(withHint("Search name or ID"))
+                .perform(typeText("Jkk"), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring("Jkk Fgh"))
+                .perform(click());
+        Thread.sleep(500);
+        //utils.ancFloatingfab()
+        //.perform(click());
+        onView(ViewMatchers.withSubstring("Call"))
+                .check(matches(isDisplayed()));
+    }
+    @Test
+    @Order(order = 5)
+    public void testANCProfileView() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)